tests/helpers/streams.ts

```diff
@@ -0,0 +1,8 @@
+export function unwrapReadableStream(stream: NodeJS.ReadableStream) {
```

Comment:
This helper allows us to unwrap readable streams to get to their contents. We need this for the server rendering tests.